db/migrate/20220813065131_create_category_rewards.rb

```diff
@@ -0,0 +1,8 @@
+class CreateCategoryRewards < ActiveRecord::Migration[6.1]
+  def change
+    create_table :category_rewards do |t|
+      t.references :category, index: true
+      t.references :reward, index: true
```

Comment:
You could add `t.timestamps` to keep it consistent with other tables (in Rails usually each table have created_at, updated_at columns). Also, maybe such columns would be useful for debugging  later on.